docs/default-firebase-data.json

```diff
@@ -728,14 +1179,17 @@
       "order": 10,
       "photoUrl": "https://media.licdn.com/dms/image/C5103AQGwVI0yKMIU8A/profile-displayphoto-shrink_200_200/0?e=1577318400&v=beta&t=V6QsnmHk-cMXWHXh9Sace7Ihos1DCslvFQrrd5AQdmw",
       "sessions": {
-        "2017": [{
-          "presentation": "https://docs.google.com/presentation/d/1ByepX6zNVZOTJnMEl01rYhaKvN-TzoeO-B-IwhAsoUs",
-          "tags": [],
-          "title": "Android Things 🛠🤖🛠",
-          "videoId": ""
-        }]
+        "2017": [
+          {
+            "presentation": "https://docs.google.com/presentation/d/1ByepX6zNVZOTJnMEl01rYhaKvN-TzoeO-B-IwhAsoUs",
+            "tags": [],
+            "title": "Android Things 🛠🤖🛠",
+            "videoId": ""
```

Comment:
Not sure if we still need this. But maybe it's just a place holder?

Reply:
Probably just a placeholder, let me check......